README.md

```
@@ -13,3 +13,33 @@ Uncomment this if you use travis:
+| disable-checkout-persist-credentials | v0.1.2 | CLI to disable actions/checkout's persist-credentials | [https://github.com/suzuki-shunsuke/disable-checkout-persist-credentials](https://github.com/suzuki-shunsuke/disable-checkout-persist-credentials) |
+| duckgo | v0.1.1 | A command line DuckDuckGo client with fuzzyfinder UI written in Go | [https://github.com/sheepla/duckgo](https://github.com/sheepla/duckgo) |
+| firge | v0.3.0 | Programming font that combines Genshin Gothic and Fira Mono | [https://github.com/yuru7/Firge](https://github.com/yuru7/Firge) |
+| firge-nerd | v0.3.0 | Programming font that combines Genshin Gothic and Fira Mono with Nerd Font | [https://github.com/yuru7/Firge](https://github.com/yuru7/Firge) |
+| ghalint | v1.5.1 | GitHub Actions linter | [https://github.com/suzuki-shunsuke/ghalint](https://github.com/suzuki-shunsuke/ghalint) |
+| ghatm | v0.3.5 | Set timeout-minutes to all GitHub Actions jobs | [https://github.com/suzuki-shunsuke/ghatm](https://github.com/suzuki-shunsuke/ghatm) |
+| ghq | v1.8.0 | Remote repository management made easy | [https://github.com/x-motemen/ghq](https://github.com/x-motemen/ghq) |
+| gitlab-ci-ls | 1.1.1 | Language server for Gitlab CI | [https://github.com/alesbrelih/gitlab-ci-ls](https://github.com/alesbrelih/gitlab-ci-ls) |
+| gitlab-ci-verify | v2.1.12 | Validate and lint your gitlab ci files using ShellCheck, the Gitlab API, curated checks or even build your own checks
+| [https://github.com/timo-reymann/gitlab-ci-verify](https://github.com/timo-reymann/gitlab-ci-verify) |
+| lazygit | v0.53.0 | simple terminal UI for git commands | [https://github.com/jesseduffield/lazygit](https://github.com/jesseduffield/lazygit) |
+| octocov | v0.68.1 | octocov is a toolkit for collecting code metrics (code coverage, code to test ratio, test execution time and your own custom metrics).
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)